lex-examples/gemini.py

```
@@ -0,0 +1,145 @@
+dump_path = Dir / "forms"
+dump_path.mkdir(exist_ok=True)
+
+file_uthmani = Dir / "quran-uthmani.json"
+if file_uthmani.exists():
+    # تحميل القرآن من الملف المحلي
+    with open(Dir / "quran-uthmani.json", "r", encoding="utf-8") as f:
+        quran_data = json.load(f)
+else:
+    # تحميل القرآن من موقع quran.com
+    url = "https://api.alquran.cloud/v1/quran/quran-uthmani"
+    response = requests.get(url)
```

Ruff (0.12.2)

Probable use of `requests` call without timeout

(S113)